src/plugins/NEMD/MettDeamon.cpp

```diff
@@ -1751,6 +1702,7 @@ void Reservoir::readFromFileBinary(DomainDecompBase* domainDecomp, ParticleConta
 		case ICRVQD: _moleculeDataReader.reset(new MoleculeDataReaderICRVQD()); break;
 		case ICRV: _moleculeDataReader.reset(new MoleculeDataReaderICRV()); break;
 		case IRV: _moleculeDataReader.reset(new MoleculeDataReaderIRV()); break;
```

Comment:
```suggestion
		case ICRVQD:
			_moleculeDataReader = std::make_unique<MoleculeDataReaderICRVQD>();
			break;
		case ICRV:
			_moleculeDataReader = std::make_unique<MoleculeDataReaderICRV>();
			break;
		case IRV:
			_moleculeDataReader = std::make_unique<MoleculeDataReaderIRV>();
			break;
```

https://stackoverflow.com/questions/41247630/should-i-use-make-unique-or-reset-for-already-defined-smart-pointers